src/Util/Reflection/PrivatesAccessor.php

```diff
@@ -47,25 +47,20 @@ public function callPrivateMethod(object|string $object, string $methodName, arr
     public function getPrivateProperty(object $object, string $propertyName): mixed
     {
         $reflectionProperty = $this->resolvePropertyReflection($object, $propertyName);
-        $reflectionProperty->setAccessible(true);
 
         return $reflectionProperty->getValue($object);
     }
 
     public function setPrivateProperty(object $object, string $propertyName, mixed $value): void
     {
         $reflectionProperty = $this->resolvePropertyReflection($object, $propertyName);
-        $reflectionProperty->setAccessible(true);
 
         $reflectionProperty->setValue($object, $value);
     }
 
     private function createAccessibleMethodReflection(object $object, string $methodName): ReflectionMethod
     {
-        $reflectionMethod = new ReflectionMethod($object, $methodName);
-        $reflectionMethod->setAccessible(true);
-
-        return $reflectionMethod;
+        return new ReflectionMethod($object, $methodName);
```

Comment:
the downgrade rule seems not handling on `return` yet, only `Expression`, I will look into it.

Reply:
Side note: `DowngradeSetAccessibleReflectionPropertyRector` has "Property" in the name. Your last PR https://github.com/rectorphp/rector-downgrade-php/pull/296 added handling for methods. Just pointing it out in case that's relevant.

Reply:
I created PR to handle on direct return

- https://github.com/rectorphp/rector-downgrade-php/pull/297